app/src/main/java/org/lineageos/tv/launcher/SystemOptionsActivity.kt

```diff
@@ -66,6 +70,8 @@ class SystemOptionsActivity : ModalActivity(R.layout.activity_system_options),
     private var notificationUpdateListener: TvNotificationListener.NotificationUpdateListener? =
         null
 
+    private val connectivityManager by lazy { getSystemService(CONNECTIVITY_SERVICE) as ConnectivityManager }
```

Comment:
`    private val connectivityManager by lazy { getSystemService(ConnectivityManager::class.java) }`
Here and elsewhere? (might be wrong, but syntax is similar)